### DISCUSSION TITLE
If the negotiated rate has ended/terminated in the monthly reporting period, does it need to be reported in that month?

### DISCUSSION THREAD

**Discussion by tdfow** [CMSgov/price-transparency-guide#40] [Q&A] [answered]
Assume that after a provider's negotiated rate has ended/terminated, it will not need to be included in the monthly files after termination.   If the termination end date is mid-month, for example---3/15 and reporting period is 3/30, does the provider's rates need to be included in the March file or can it be excluded?

**Comment by shaselton-usds** [comment, marked as answer]
Great question! I will review with the team and get back to you. 

My knee-jerk reaction is "yes, they would be included". This is because the reporting is meant to reflect the previous month's snap shot in time. There is potentially a future where this data would be required to be more real-time, but this is the first iterative step.

**Reply by tdfow** [reply]
@shaselton, do you have any updates on this?   Is it still, yes, include if they were effective any time during the reporting period or only needed to report if effective on the data the information is pulled?

**Reply by shaselton-usds** [reply]
@tdfow Thanks for the patience.  If there was a contract that was active at any time during the reporting period, it would need to be reported.

**Reply by tdfow** [reply]
@shaselton-usds, Thanks, as we wanted to exclude any providers that were not applicable to report.

**Comment by ghost** [comment]
Good question because for some of the more complicated negotiated rates, some plans will need to generate dummy claims to provide the necessary negotiated rate for a provider per plan. The dummy claims will need to have a date of service (DOS) to be adjudicated and could fall out if the based on your mid-month example.

**Reply by shaselton-usds** [reply]
Are you pulling this information on negotiated contracts for items/services from claims data?

**Reply by ghost** [reply]
@shaselton-usds No we are running based on active plans, active in-network associated providers using a real time claim adjudicator api which requires a DOS.
